app/code/Magento/Catalog/Test/Mftf/Section/StorefrontCategoryBottomToolbarSection.xml

```diff
@@ -12,6 +12,6 @@
         <element name="previousPage" type="button" selector=".//*[@class='toolbar toolbar-products'][2]//a[contains(@class, 'previous')]" timeout="30"/>
         <element name="pageNumber" type="text" selector="//*[@class='toolbar toolbar-products'][2]//a[contains(@class, 'page')]//span[2][contains(text() ,'{{var1}}')]" parameterized="true"/>
         <element name="perPage" type="select" selector="//*[@class='toolbar toolbar-products'][2]//select[@id='limiter']"/>
-        <element name="currentPage" type="text" selector=".products.wrapper + .toolbar-products .pages .current span:nth-of-type(2)"/>
+        <element name="currentPage" type="text" selector=".//*[@class='toolbar toolbar-products'][2]//li[contains(@class, 'current')]//span[2]" timeout="30"/>
```

Comment:
Is there any other way - without using [n] to point the bottom pagination? Some can just change the top one and... The selector won't work for bottom one.

Reply:
I don't think there is any reliable way to do that. Changing it into `.products.wrapper + .toolbar-products` as it was before has even more chance to fail, as it's relatively easy to insert something between products and bottom toolbar, but moving or removing one of toolbars is one of things that are currently impossible - https://github.com/magento/magento2/issues/7253 🙂